src/BuildScriptGenerator/DotNetCore/DotnetCoreConstants.cs

```diff
@@ -8,7 +8,8 @@ namespace Microsoft.Oryx.BuildScriptGenerator.DotNetCore
     public static class DotnetCoreConstants
     {
         public const string LanguageName = "dotnet";
-        public const string ProjectFileExtensionName = "csproj";
+        public const string CSharpProjectFileExtensionName = "csproj";
+        public const string FSharpProjectFileExtensionName = "fsproj";
```

Comment:
I'd argue that `CSharpProjectFileExtension`/`FSharpProjectFileExtension` are sufficient (no need for the `Name` suffix)